src/Service/Services/MetadataProviders/CosmosSqlMetadataProvider.cs

```diff
@@ -196,12 +205,25 @@ public List<string> GetSchemaGraphQLFieldNamesForEntityName(string entityName)
             return new List<string>();
         }
 
-        public string? GetSchemaGraphQLFieldTypeByFieldName(string graphQLType, string fieldName)
+        /// <summary>
+        /// Give an entity name and its field name, 
+        /// this method is to first look up the GraphQL field type using the entity name,
+        /// then find the field type with graphQL type and its field name.
+        /// </summary>
+        /// <param name="graphQLType">GraphQL type</param>
+        /// <param name="fieldName">GraphQL field name</param>
+        /// <returns></returns>
```

Comment:
These look to be outdated due to more recent changes.

Reply:
Updated, thank you for catching this.